samples/client/petstore/kotlin-retrofit2-kotlinx_serialization/src/main/kotlin/org/openapitools/client/models/Pet.kt

```diff
@@ -67,6 +67,7 @@ data class Pet (
      * Values: AVAILABLE,PENDING,SOLD
      */
     @KSerializable
+    
```

Comment:
Is there a way to skip this empty line, when not moshi is used as a serialisation library?

Reply:
I left a tip to fix this in a comment I opened.